v3/ecl/vna/v1/appliances/testing/fixtures.go

```diff
@@ -514,6 +514,10 @@ var createRequest = fmt.Sprintf(`
     				"network_id": "dummyNetworkID"
     			}
     		},
+            "initial_config": {
+                "format": "set",
+                "data": "c2V0IGludGVyZmFjZXMgZ2UtMC8wLzAgZGVzY3JpcHRpb24gc2FtcGxl"
+            },
```

Comment:
```suggestion
    		"initial_config": {
    			"format": "set",
    			"data": "c2V0IGludGVyZmFjZXMgZ2UtMC8wLzAgZGVzY3JpcHRpb24gc2FtcGxl"
    		},
```
他の行にインデントを合わせていただけますか。
（そもそも先頭だけ半角スペースにしている他の行のインデントの仕方が悪いのですが……）

Reply:
コメントありがとうございます。
スペースと混ざっているの気づいてませんでした。
他と統一済みです。